examples/data_preprocessing/mimic_dataset/prepare_mimic_cxr.ipynb

```diff
@@ -0,0 +1,1520 @@
+       "4          NaN           NaN            1.0    NaN   \n",
+       "5          NaN           NaN            NaN    NaN   \n",
+       "6          NaN           NaN            NaN    NaN   \n",
+       "7          NaN           NaN            NaN   -1.0   \n",
+       "\n",
+       "   Enlarged Cardiomediastinum  Fracture  Lung Lesion  Lung Opacity  \\\n",
+       "0                         NaN       NaN          NaN           NaN   \n",
+       "1                         NaN       NaN          NaN           NaN   \n",
+       "2                         NaN       NaN          NaN           NaN   \n",
+       "3                         NaN       NaN          NaN           NaN   \n",
+       "4                         NaN       NaN          NaN           NaN   \n",
+       "5                         NaN       NaN          NaN           NaN   \n",
+       "6                         NaN       NaN          NaN           NaN   \n",
+       "7                         NaN       NaN          NaN          -1.0   \n",
+       "\n",
+       "   No Finding  Pleural Effusion  Pleural Other  Pneumonia  Pneumothorax  \\\n",
+       "0         1.0               NaN            NaN        NaN           NaN   \n",
+       "1         1.0               NaN            NaN        NaN           NaN   \n",
+       "2         1.0               NaN            NaN        NaN           NaN   \n",
+       "3         1.0               NaN            NaN        NaN           NaN   \n",
+       "4         NaN               NaN            NaN       -1.0           NaN   \n",
+       "5         1.0               NaN            NaN        NaN           NaN   \n",
+       "6         1.0               NaN            NaN        NaN           NaN   \n",
+       "7         NaN               1.0            NaN        1.0           NaN   \n",
+       "\n",
+       "   Support Devices  \n",
+       "0              NaN  \n",
+       "1              NaN  \n",
+       "2              NaN  \n",
+       "3              NaN  \n",
+       "4              NaN  \n",
+       "5              NaN  \n",
+       "6              NaN  \n",
+       "7              NaN  "
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "split_list = pd.read_csv(\"mimic-cxr-2.0.0-split.csv\")\n",
+    "labels_chexpert = pd.read_csv(\"mimic-cxr-2.0.0-chexpert.csv\")\n",
+    "\n",
+    "labels_chexpert.iloc[:,2:].head(8) #note that we removed the ids in the output due to data privacy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To get one label for each study, we do the following:\n",
+    "If there is exactly one label positively mentioned, this label will be assigned to the study. If there are multiple labels positively mentioned, one label will be chosen randomly. When there is no 1.0 assigned to the study, the label will be set to 'No Finding'."
```

Comment:
if you are transforming the data to Knodle input, why would you need labels?

Reply:
I use the labels for the image encoding (second approach including finetuning).